students/FlyStriker/dom-practice/README.md

```diff
@@ -0,0 +1,2 @@
+# dom-practice
+https://flystriker.github.io/dom-practice/
```

Comment:
make it more handy with [Markdown links syntax](https://anvilproject.org/guides/content/creating-links)